includes.mk

```diff
@@ -0,0 +1,11 @@
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+		echo "Missing \`docker\` client which is required for development"; \
+		exit 2; \
+	fi
+
+check-kubectl:
+	@if [ -z $$(which kubectl) ]; then \
```

Comment:
what if `which` prints something out? This might be more accurate to what we're looking for: `ifeq ($(shell which kubectl >/dev/null 2>&1; echo $$?), 1)`